pkg/sink/heroku.go

```diff
@@ -0,0 +1,61 @@
+package sink
+
+import (
+	"context"
+	"fmt"
+
+	heroku "github.com/heroku/heroku-go/v5"
+	"github.com/pkg/errors"
+)
+
+type HerokuServiceIface interface {
+	ConfigVarUpdate(ctx context.Context, appIdentity string, o map[string]*string) (heroku.ConfigVarUpdateResult, error)
+	ConfigVarInfoForApp(ctx context.Context, appIdentity string) (heroku.ConfigVarInfoForAppResult, error)
+}
+
+type HerokuSink struct {
+	BaseSink    `yaml:",inline"`
+	Client      HerokuServiceIface `yaml:"client"`
+	AppIdentity string             `yaml:"AppIdentity"`
+}
+
+func (sink *HerokuSink) WithHerokuClient(client HerokuServiceIface) *HerokuSink {
+	sink.Client = client
+	return sink
+}
+
+func (sink *HerokuSink) GetKeyToName() map[string]string {
+	return sink.KeyToName
+}
+
+func (sink *HerokuSink) WithKeyToName(m map[string]string) *HerokuSink {
+	sink.BaseSink = BaseSink{KeyToName: m}
+	return sink
+}
+
+// Write writes the value of the env var with the specified name for the given repo
+func (sink *HerokuSink) Write(ctx context.Context, name string, val string) error {
+
+	keypair := map[string]*string{
```

Comment:
```suggestion
	varUpdates := map[string]*string{
```
keypair usually associated with public/private crypto